services/simcore-agent/setup.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import re
+import sys
+from pathlib import Path
+
+from setuptools import find_packages, setup
+
+
+def read_reqs(reqs_path: Path) -> set[str]:
+    return {
+        r
+        for r in re.findall(
+            r"(^[^#\n-][\w\[,\]]+[-~>=<.\w]*)",
+            reqs_path.read_text(),
+            re.MULTILINE,
+        )
+        if isinstance(r, str)
+    }
+
+
+CURRENT_DIR = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
+
+NAME = "simcore-service-simcore-agent"
+VERSION = (CURRENT_DIR / "VERSION").read_text().strip()
+AUTHORS = ("Andrei Neagu (GitHK)",)
+DESCRIPTION = 'Service for executing commands on docker nodes'
+README = (CURRENT_DIR / "README.md").read_text()
+
+PROD_REQUIREMENTS = tuple(
+    read_reqs(CURRENT_DIR / "requirements" / "_base.txt")
+    | {
+        "simcore-models-library",
+        "simcore-settings-library",
+    }
+)
+
+TEST_REQUIREMENTS = tuple(read_reqs(CURRENT_DIR / "requirements" / "_test.txt"))
+
+SETUP = dict(
+    name=NAME,
+    version=VERSION,
+    author=AUTHORS,
+    description=DESCRIPTION,
+    long_description=README,
+    license="MIT license",
+    python_requires="~=3.9",
+    packages=find_packages(where="src"),
+    package_dir={
+        "": "src",
+    },
+    include_package_data=True,
+    install_requires=PROD_REQUIREMENTS,
+    test_suite="tests",
+    tests_require=TEST_REQUIREMENTS,
+    extras_require={"test": TEST_REQUIREMENTS},
+    entry_points={
+        "console_scripts": [
+            "simcore-service-simcore-agent = simcore_service_simcore_agent.cli:main",
```

Comment:
these names are horrible :-)))
If you are so keen on simply calling it ``simcore-agent``  (i still find it too general though ... ) I would drop the ``simcore`` since it is already implicit in all the "mandatory" prefix. Therefore:

- folder would be ``osparc-simcore/services/agent``
- the python package would be called ``simcore_service_agent``
- and the entrypoint would be ``simcore-service-agent``

Reply:
this was renamed to agent